src/components/MarketPlace/MarketPlace.jsx

```diff
@@ -127,27 +132,55 @@ export default function MarketPlace(props) {
         return `${quantityWithUnits} ${unit}`
     }
 
+    const getSellerOrder = (curUsers) => {
+        let sellerNumProducts = curUsers.map((user, indx) => {
+            if (user.uid != currentUser.uid && user.data.sale) {
```

Comment:
this could probably be a .filter as well to make this simpler

Reply:
note to talk about this during our 1:1